v2-components/main-action-card/MainActionCard.tsx

```diff
@@ -0,0 +1,235 @@
+import { Badge, Box, Button, Flex, Heading, Stack, Text } from '@chakra-ui/react';
+import React, { PropsWithChildren, ReactNode } from 'react';
+import { getVariant } from '../c-ratio-health-card/getVariant';
```

Comment:
This function is probably gonna be important in a few places. Maybe we should think about a better name and put it elsewhere.

Reply:
Yeah for sure. Maybe something like solvency/health state?

Reply:
Just keep it in v2-components for now?  Maybe `getHealthVariant` since we have color and components variants defined with these values?